python/lsst/ts/mtrotator/rotator_csc.py

```diff
@@ -459,8 +470,23 @@ async def do_move(self, data: salobj.BaseMsgType) -> None:
             force_output=True,
         )
 
+    def assert_is_motion_locked(self) -> None:
+        """Assert the motion is locked.
+
+        Raises
+        ------
+        `salobj.ExpectedError`
+            When the motion is locked.
+        """
+
+        if self._is_motion_locked:
+            raise salobj.ExpectedError("Motion is locked")
```

Comment:
It is a bit weird that an `assert_...` method raises something other than an `AssertionError` no?

I suggest you change this to something like:


```
assert self._is_motion_locked, "Motion is locked."
```

or, if you prefer the more verbose option:

```
if self._is_motion_locked:
    raise AssertionError("Motion is locked.")
```